templates/my_master.html

```diff
@@ -30,7 +30,12 @@
                 <li>merchant: <code>{{merchant_rate * 100}}%</code></li>
                 <li>customer: <code>{{customer_rate * 100}}%</code></li>
             </ul>
-            Settlement address: <code>{{settlement_address}}</code>
+        </small>
```

Comment:
better to merge these two blocks so that admin/finance both see everything (rates, settlement address and balance)